lua/ulx/modules/sh/freeze.lua

```diff
@@ -0,0 +1,36 @@
+CATEGORY_NAME = "CFC"
+
+local function freezeProps( callingPlayer, targetPlayers ) 
+    local entities = ents.GetAll()
+    local entCount = 0
+    local entCounts = {}
+    for _, ply in pairs( targetPlayers ) do
+        entCounts[ply] = 0
+    end
+
+    for _, ent in pairs( entities ) do
+        local owner = ent:CPPIGetOwner()
+        if entCounts[owner] then
+            local canFreeze = not (ent:IsWeapon() or ent:GetUnFreezable() or ent:IsPlayer())
+            local physicsObj = ent:GetPhysicsObject()
+            if IsValid( physicsObj ) and canFreeze then
+                physicsObj:EnableMotion( false )
+                physicsObj:Sleep()
+                entCount = entCount + 1
+                entCounts[owner] = entCounts[owner] + 1
+            end
+        end
+    end
+    local concatVal = ""
+    for nam, num in pairs( entCounts ) do
```

Comment:
`nam`? if this is meant to be name just write `name`. although `ply` is probably a more appropriate variable name.

Reply:
num nam tim tam